Gemfile.lock

```diff
@@ -70,29 +71,28 @@ GEM
       net-imap
       net-pop
       net-smtp
-    mini_mime (1.1.2)
+    mini_mime (1.1.5)
     mini_portile2 (2.8.7)
-    minitest (5.23.1)
+    minitest (5.25.0)
     mutex_m (0.2.0)
-    net-imap (0.3.4)
+    net-imap (0.4.14)
       date
       net-protocol
     net-pop (0.1.2)
       net-protocol
-    net-protocol (0.2.1)
+    net-protocol (0.2.2)
       timeout
-    net-smtp (0.3.3)
+    net-smtp (0.5.0)
       net-protocol
     nokogiri (1.15.6)
       mini_portile2 (~> 2.8.2)
```

Comment:
Could we bump nokogiri too? https://github.com/railsware/actionmailer-balancer/security/dependabot/30

Reply:
Damn me! I did that but because I decided to keep the older ruby version support later on the change got overwritten. Thanks for bringing that up!